tests/idris2/basic/basic044/expected

```diff
@@ -117,33 +117,33 @@ Target type : ({arg:1} : (Data.Fin.Fin (Prelude.Types.S (Prelude.Types.S Prelude
 LOG elab.ambiguous:5: Ambiguous elaboration (kept 2 out of 2 candidates) (not delayed) at Vec:3:
 $resolved6
 $resolved7
-Target type : ?Vec.{a:4569}_[]
+Target type : ?Vec.{a:4704}_[]
```

Comment:
In testutils.sh we have tools to get rid of these machine names which are changing for no good reason
when the libs are modified. Please use that rather than updating the golden file